test/bin.js

```diff
@@ -240,12 +240,15 @@ test('lint command', function (group) {
 
   group.test('generates lint output', options, function (t) {
     documentation(['lint fixture/lint/lint.input.js'], function (err, data) {
-      var output = path.join(__dirname, 'fixture/lint/lint.output.js');
+      var output = path.join(__dirname, 'fixture', 'lint', 'lint.output.js');
       data = data.toString().split('\n').slice(2).join('\n');
       if (process.env.UPDATE) {
         fs.writeFileSync(output, data);
       }
       t.equal(err.code, 1);
+      if (os.type() === 'Windows_NT') {
+        data = data.replace('‼', '⚠');
```

Comment:
Making a note of why this required a workaround: vfile-reporter uses log-symbols https://github.com/sindresorhus/log-symbols and log-symbols uses different symbols for Windows's limited character set than for other platforms with broader UTF8 support.